n3fit/src/n3fit/model_gen.py

```diff
@@ -565,7 +565,7 @@ def pdfNN_layer_generator(
 
     # Normalization and sum rules
     if impose_sumrule:
-        sumrule_layer, integrator_input = msr_impose(mode=impose_sumrule, scaler=scaler)
+        sumrule_layer, integrator_input = msr_impose(basis_size=last_layer_nodes, mode=impose_sumrule, scaler=scaler)
```

Comment:
```suggestion
        sumrule_layer, integrator_input = msr_impose(mode=impose_sumrule, scaler=scaler)
```

That `basis_size` argument remained there but it is no longer used (even if it is passed to the layer). Should've been removed from everywhere ages ago.

Reply:
ok I was gonna ask. I ll remove t

Reply:
Since you're at it remove it also from `msr_impose` please! Thanks!